website/docs/components/table/advanced-table/partials/code/how-to-use.md

```diff
@@ -594,15 +594,15 @@ Note: Using `@hasStickyFirstColumn` with nested rows is not supported.
 
 ### Multi-select Advanced Table
 
-!!! Warning
+A multi-select Advanced Table includes checkboxes enabling users to select multiple rows for purposes of performing bulk operations. Checking or unchecking the checkbox in the Advanced Table header either selects or deselects the checkboxes on each row in the body. Individual checkboxes in the rows can also be selected or deselected.
 
-At this time, the Advanced Table does not support multi-select nested rows. If this is a use case you require, please [contact the Design Systems Team](/about/support).
+Add `isSelectable=true` to create a multi-select Advanced Table. The `onSelectionChange` argument can be used to pass a callback function to receive selection keys when the selected rows change. You must also pass a `selectionKey` to each row which gets passed back through the `onSelectionChange` callback which maps the row selection on the Advanced Table to an item in your data model.
 
-!!!
+!!! Callout
```

Comment:
Should this Callout banner be moved below the `### Multi-select Advanced Table` header?